src/vs/workbench/contrib/terminal/browser/terminalInstance.ts

```diff
@@ -2694,6 +2694,19 @@ export function parseExitResult(
 	return { code, message };
 }
 
+const introMessage = nls.localize('introMsg', "Welcome to Terminal Accessibility Help");
+const enterAccessibilityModeNls = nls.localize('enterAccessibilityMode', 'The Enter Accessibility Mode ({0}) command enables screen readers to read terminal contents.');
+const enterAccessibilityModeNoKb = nls.localize('enterAccessibilityModeNoKb', 'The Enter Accessibility Mode command enables screen readers to read terminal contents and is currently not triggerable by a keybinding.');
+const shellIntegration = nls.localize('shellIntegration', "The terminal has a feature called shell integration that offers an enhanced experience and provides useful commands for screen readers such as:");
+const runRecentCommand = nls.localize('runRecentCommand', 'Run Recent Command ({0})');
+const runRecentCommandNoKb = nls.localize('runRecentCommandNoKb', 'Run Recent Command is currently not triggerable by a keybinding.');
+const goToRecent = nls.localize('goToRecentDirectory', 'Go to Recent Directory ({0})');
+const goToRecentNoKb = nls.localize('goToRecentDirectoryNoKb', 'Go to Recent Directory is currently not triggerable by a keybinding.');
+const readMore = nls.localize('readMore', 'Read more about terminal accessibility at https://code.visualstudio.com/docs/terminal/shell-integration.');
+const dismiss = nls.localize('dismiss', "You can dismiss this dialog by pressing Escape or focusing elsewhere.");
+const openDetectedLink = nls.localize('openDetectedLink', 'The Open Detected Link ({0}) command enables screen readers to easily open links found in the terminal.');
```

Comment:
I think using `{0}` in an nls call meant for string.format could confuse the tooling? It's unexpected from my perspective

Reply:
I'm doing what the editor does here, doesn't seem like it causes problems